exonum-java-binding-core/rust/src/runtime/cmd.rs

```diff
@@ -22,12 +22,12 @@ impl CommandExtension for GenerateNodeConfig {
     fn args(&self) -> Vec<Argument> {
         vec![
             Argument::new_named(
-                EJB_DEBUG,
+                EJB_USER_PARAMETERS,
                 false,
-                "Debug mode for JVM.",
+                "Additional parameters for JVM.",
```

Comment:
Will it be shown in help? If so, I'd document that these parameters must not have a leading dash (`-`), e.g., `Xmx2G`.